ztp/siteconfig-generator/siteConfig/siteConfig.go

```diff
@@ -189,6 +189,7 @@ type Nodes struct {
 	InstallerArgs          string                 `yaml:"installerArgs"`
 	IgnitionConfigOverride string                 `yaml:"ignitionConfigOverride"`
 	Role                   string                 `yaml:"role"`
+	BiosConfigMapRef       BiosConfigMapRef       `yaml:"biosConfigMapRef"`
```

Comment:
Would you also define this to the cluster level, So if the cluster has same nodes don't need to keep define it per node.
AND the node definition override the cluster definition if both defined.

Reply:
Yes, thanks for the suggestion. I will add BiosConfigMapRef to the cluster level as well.